oeps/architectural-decisions/oep-0064-mobile-application-codebase-modernization.rst

```diff
@@ -0,0 +1,103 @@
+* https://github.com/raccoongang/educationx-app-ios
+* https://github.com/raccoongang/educationx-app-android
+
+* The new repositories will be migrated to the openedx github organization under the name openedx-app-ios and openedx-app-android.
+* The new repositories will be maintained by the Racoon Gang team with Marco Morales serving as the product lead for the project.
+* Relevant funded contributions and projects relation to the new mobile repositories will be linked from their main project readme as a way to help community members find ongoing and future efforts
+
+
+Rationale
+*********
+
+Expanded access and opportunities for learners around the globe means meeting learners where they are, and for many people that means on their only digital device, their phone. Furthermore, enabling learners to continue to learn on their desktop or laptop computers in addition to a full featured mobile experience allows us to support full day learning workflows for those with multiple devices.
+
+Similar to the way the edx-platform has undergone a modernization process over the past few years with updated build processes, micro services, micro frontends, design system tools and more, the mobile experiences will benefit from a modern architecture, being written in the latest development language tools, and set up for increased pluggability and contribution as well.
+
+Backward Compatibility
+**********************
+The existing mobile applications will continue to be supported by edX / 2U for the foreseeable future. We do not yet have a clear timeline for when edX / 2U might switch over to the new applications, but their focus is on rollout and delivery of their latest improvements to the existing experience including the commerce capabilities that have been added recently. 
+
+Reference Implementation
+************************
+
+The new mobile repositories are maintained by Raccoon Gang and can be accessed here:
+
+* https://github.com/raccoongang/educationx-app-ios
+* https://github.com/raccoongang/educationx-app-android
+
+Rejected Alternatives
+*********************
+
+We considered exploring the development costs of incrementally improving the existing mobile applications to fully adopt new MVVM patterns as well as the use of Swift and Kotlin. Efforts to use these new technologies go back many years, but it would take considerable effort for the community to support this migration. Realistically the current team at edX / 2U maintaining the repositories would have had to undertake a major development effort to achieve this transformation. 
+While Raccoon Gang considered charting a path to modernizing the existing code bases, it took about 3-4 months worth of development from two team members on new architectural patterns and programming languages to develop the new mobile applications, demonstrating major improvements to development speed with the new repository option.
```

Comment:
How is this a rejected alternative?

Reply:
removed this - it was meant originally to convey that the proof of concept new apps demonstrated the viability of the other direction, but its an unnecessary detail for this. removed!

Reply:
@marcotuts there's one comment above which I'm not certain was addressed, https://github.com/openedx/open-edx-proposals/pull/496#discussion_r1262780883  If it is, I'll resolve this conversation so we can merge this PR.

Reply:
Yup, ready to resolve this section was removed based on the broader feedback of OEP scope / not calling out details that may change elsewhere etc. The read me for the legacy mobile apps is the place to understand active maintenance / other details.